hw33springbeans/src/main/java/ua/ithillel/beans/Cart.java

```diff
@@ -0,0 +1,40 @@
+package ua.ithillel.beans;
+
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.context.annotation.Scope;
+import org.springframework.context.annotation.ScopedProxyMode;
+import org.springframework.stereotype.Component;
+import ua.ithillel.entities.Product;
+import ua.ithillel.interfaces.CartOps;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+@Scope(value = ConfigurableBeanFactory.SCOPE_PROTOTYPE, proxyMode = ScopedProxyMode.DEFAULT)
+public class Cart implements CartOps {
+
+    private final List<Product> cartContent = new ArrayList<>();
+
```

Comment:
private final ProductRepository productRepository. And constructor for it (for autowiring)